FrameworklessServerTests/UsersServiceTests.cs

```diff
@@ -111,5 +105,27 @@ public void GivenUserNameShouldReturnNameFromList()
             Assert.Equal(expected.Name, result.Name);
             ResetList();
         }
+
+        [Fact]
+        public void GivenUserNameShouldUpdateToNewName()
+        {
+            _usersService.Add(new User("Bob"));
+            var expected = new List<User> {new User("Cindy"), new User("Bob")};
+            Assert.Equal(expected[1].Name, _usersService.GetAllUsers()[1].Name);
+
+            _usersService.UpdateUser("Bob", "Mary");
+            
+             expected = new List<User> {new User("Cindy"), new User("Mary")};
+            Assert.Equal(expected[1].Name, _usersService.GetAllUsers()[1].Name);
+            ResetList();
+        }
+
+        [Fact]
+        public void GivenNullUserNameWorldOwnerNameOrNonExistentNameShouldThrowException()
+        {
+            Assert.Throws<InvalidOperationException>(() => _usersService.UpdateUser("Sue", "Mary"));
+            Assert.Throws<InvalidOperationException>(() => _usersService.UpdateUser(null, "Mary"));
+            Assert.Throws<InvalidOperationException>(() => _usersService.UpdateUser("Cindy", "Mary"));
```

Comment:
Whats the diff between this assert and the first one? How can you make that diff stand out instead of me having to calculate it in my head?